packages/components/src/toolbar/toolbar-group/style.scss

```diff
@@ -1,7 +1,6 @@
 .components-toolbar-group {
 	min-height: $block-toolbar-height;
 	border-right: $border-width solid $gray-900;
-	background-color: $white;
```

Comment:
This feels like something unintended. Should it be part of another PR? I know we're overriding the background color here and there, so we might want to audit those instances separately, for example:

https://github.com/WordPress/gutenberg/blob/base-control-margin/packages/block-editor/src/components/block-toolbar/style.scss#L26

Reply:
The change is intended; without it, the group background obscures the radius of the container:

<img width="826" alt="Screenshot 2024-08-12 at 20 21 47" src="https://github.com/user-attachments/assets/136ddcee-723b-4d25-90e6-1c16a6c71e04">

We could add a radius to the group instead of removing the background, but that causes issues with the separator styling:

<img width="807" alt="Screenshot 2024-08-12 at 20 22 45" src="https://github.com/user-attachments/assets/a9201815-15e2-4388-8be3-533522b9abc7">

I don't mind extracting the toolbar change from this PR and updating it separately though. In fact that's probably a good idea given it's likely the most impactful change. We can discuss the intricacies in a different PR.

Reply:
Yeah, I'd prefer a separate PR if that's possible. But I'll leave that call to you, it doesn't feel like this is a blocker for this PR anyway.